core/src/main/java/com/datastax/oss/driver/internal/core/metadata/LoadBalancingPolicyWrapper.java

```diff
@@ -180,14 +181,17 @@ private void processNodeStateEvent(NodeStateEvent event) {
         return; // ignore
       case RUNNING:
         for (LoadBalancingPolicy policy : policies) {
-          if (event.newState == NodeState.UP) {
-            policy.onUp(event.node);
+          DefaultNode node = event.node.get();
+          if (node == null) {
```

Comment:
nit: do we need to re-perform the null for every policy? Is there a good reason not to pull this out of the loop?